pkg/client/drive.go

```diff
@@ -23,6 +23,7 @@ import (
 	"strings"
 
 	directcsi "github.com/minio/directpv/pkg/apis/direct.csi.min.io/v1beta3"
+	"github.com/minio/directpv/pkg/mount"
```

Comment:
Send  `pkg/sys/mount` -> `pkg/mount` changes as separate PR and have this PR depends on that.